app/components/Moderation/Moderation.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { connect } from "react-redux"
+import { translate } from 'react-i18next'
+
+import ReputationGuard from '../ReputationGuard'
+
+import { fetchRandomModeration, postModerationFeedback } from '../../state/moderation/effects'
+import { removeItem } from '../../state/moderation/reducer'
+import Entity from '../UsersActions/Entity'
+import ModerationEntry from './ModerationEntry'
+
+import { MODERATION_REPUTATION_REQUIRED } from '../../constants'
+
+@connect(state => ({
+  isLoading: state.Moderation.isLoading,
+  error: state.Moderation.error,
+  items: state.Moderation.items
+}), { fetchRandomModeration, postModerationFeedback })
+@translate('moderation')
+export default class Moderation extends React.PureComponent {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { isLoading, error, items, t } = this.props
+
+    return (
+      <ReputationGuard requiredRep={MODERATION_REPUTATION_REQUIRED}>
+        <div className="container">
+          <h1 className="title is-1 has-text-centered">{t('title')}</h1>
+          {items && items.map(el =>
+            <ModerationEntry key={el.id} entry={el} onAction={this.onEntryAction.bind(this)}></ModerationEntry>)
```

Comment:
j'ai un doute sur le bind(this), il n'y a pas plus simple ?

Reply:
Les deux approches possibles :

* Quand la fonction n'est appelée qu'à un seul endroit :

```
onAction={(e, a) => this.onEntryAction(e, a)}
```

* Sinon le bind dans le constructor pour ne le faire qu'une fois :

```
constructor(props) {
  super(props)
  this.onEntryAction = this.onEntryAction.bind(this)
}
```

Mais ton approche fonctionne, c'est pas un problème